examples/PySDM_examples/Grabowski_and_Pawlowska_2023/simulation.py

```diff
@@ -59,9 +59,6 @@ def __init__(
         attributes["kappa times dry volume"] = np.append(
             attributes["kappa times dry volume"], v_dry * kappa
         )
-        attributes["kappa"] = np.append(
-            attributes["kappa"], np.full(settings.n_sd, kappa)
-        )
```

Comment:
no need for `kappa` because we have `kappa times dry volume`?

Reply:
yes